src/PaymentGateways/Gateways/Stripe/StripePaymentElementGateway/stripePaymentElementGateway.tsx

```diff
@@ -88,12 +92,14 @@ interface StripeGateway extends Gateway {
 const stripePaymentElementGateway: StripeGateway = {
     id: 'stripe_payment_element',
     initialize() {
-        const {stripeKey, stripeConnectedAccountId} = this.settings;
+        const {stripeKey, stripeConnectedAccountId, formId} = this.settings;
 
         if (!stripeKey || !stripeConnectedAccountId) {
             throw new Error('Stripe gateway settings are missing.  Check your Stripe settings.');
         }
 
+        appearanceOptions = applyFilters('givewp_stripe_payment_elements_appearance_options', {}, formId) as object;
+
```

Comment:
I know this is nit-picky but the gateway id is `stripe_payment_element` - so if we're trying to reference the gateway ID here it should match 😅

```suggestion
        appearanceOptions = applyFilters('givewp_stripe_payment_element_appearance_options', {}, formId) as object;

```

Reply:
All good, thanks!